content/docs/tutorials/getting-started-aks-letsencrypt/README.md

```diff
@@ -0,0 +1,812 @@
+> Older versions of cert-manager support other authentication mechanisms which are not covered in this tutorial.
+>
+> 📖 Read about [other ways to configure the ACME issuer with Azure DNS](../../configuration/acme/dns01/azuredns/README.md).
+
+## Install the Azure workload identity features
+
+The workload identity features in Azure AKS are relatively new (at time of writing) and they require some non-default features to be enabled.
+
+Install the [Azure CLI AKS Preview Extension](https://github.com/Azure/azure-cli-extensions/tree/main/src/aks-preview),
+which you will need to configure some advanced workload identity federation features on your AKS cluster.
+
+```bash
+az extension add --name aks-preview
+```
+
+Register the `EnableWorkloadIdentityPreview` feature flag which is required for the AKS cluster in this demo.
+
+```bash
+az feature register --namespace "Microsoft.ContainerService" --name "EnableWorkloadIdentityPreview"
+
+# It takes a few minutes for the status to show Registered. Verify the registration status by using the az feature list command:
+az feature list -o table --query "[?contains(name, 'Microsoft.ContainerService/EnableWorkloadIdentityPreview')].{Name:name,State:properties.state}"
+
+# When ready, refresh the registration of the Microsoft.ContainerService resource provider by using the az provider register command:
+az provider register --namespace Microsoft.ContainerService
+```
+
+> 📖 Read more about [Registering the `EnableWorkloadIdentityPreview` feature flag](https://learn.microsoft.com/en-us/azure/aks/workload-identity-deploy-cluster).
+
+## Reconfigure the cluster
+
+Next enable the workload identity federation features on the cluster that you created earlier:
+
+```bash
+az aks update \
+    --name ${CLUSTER} \
+    --enable-oidc-issuer \
+    --enable-workload-identity # ℹ️ This option is currently only available when using the aks-preview extension.
+```
+
+> 📖 Read [Deploy and configure workload identity on an Azure Kubernetes Service (AKS) cluster](https://learn.microsoft.com/en-us/azure/aks/workload-identity-deploy-cluster) for more information about the `--enable-workload-identity` feature.
+
+## Reconfigure cert-manager
+
+We will label the cert-manager controller Pod and ServiceAccount for the attention of the Azure Workload Identity webhook,
+which will result in the cert-manager controller Pod having an extra volume containing a Kubernetes ServiceAccount token which it will use to authenticate with Azure.
+
+The labels can be configured using the Helm values file below:
+
+```yaml
+# values.yaml
+podLabels:
+  azure.workload.identity/use: "true"
+serviceAccount:
+  labels:
+    azure.workload.identity/use: "true"
+```
+🔗 <a href="values.yaml">`values.yaml`</a>
+
+```
+helm upgrade cert-manager jetstack/cert-manager \
+    --namespace cert-manager \
+    --reuse-values \
+    --values values.yaml
```

Comment:
nit (non-blocking): missing a tag for the code block, but it doesn't really matter!

Reply:
Done https://github.com/cert-manager/website/pull/1140